api/database/seeders/DatabaseSeeder.php

```diff
@@ -189,6 +190,8 @@ private function seedPoolCandidate(User $user, Pool $pool)
             // match arrays from the user
             $candidate->expectedClassifications()->sync($user->expectedClassifications()->pluck('classifications.id')->toArray());
             $candidate->cmoAssets()->sync($user->cmoAssets()->pluck('cmo_assets.id')->toArray());
+
+            $candidate->createSnapshot();
```

Comment:
minor quibble, but could this run provided `submitted_at != null`? mostly just for closer approximation to expected production results

Reply:
This has been updated with [only seed candidate snapshot if submitted](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3957/commits/dd9cbcd0dffcf8c91d50ec3d6cc7108be0398625)